.github/workflows/building.yaml

```diff
@@ -52,3 +52,13 @@ jobs:
           name: econnect-alarm-${{ env.PKG_VERSION }}.tar.gz
           path: dist/econnect_alarm-${{ env.PKG_VERSION }}.tar.gz
           if-no-files-found: error
+
+      - name: ZIP Component Dir
+        run: |
+          cd ${{ github.workspace }}/custom_components/econnect_alarm
+          zip econnect_alarm.zip -r ./
+
+      - name: Upload zip to release
+        uses: softprops/action-gh-release@v0.1.15
+        with:
+          files: ${{ github.workspace }}/custom_components/econnect_alarm/econnect_alarm.zip
```

Comment:
instead of calling the file `econnect_alarm.zip` we may call it `hacs_econnect_alarm.zip`. Just to mention people that this is a HACS installer package. What do you think?